bin/ams-consumerd

```diff
@@ -34,6 +33,9 @@ def doMainProgram():
     pullInterval = config.getOption(AmsConsumerConfig.AMS, 'PullIntervalSec')
     subscriptions = config.getOption(AmsConsumerConfig.AMS, 'Subscriptions')
     msgNum = config.getOption(AmsConsumerConfig.AMS, 'PullMsgs')
+
+    global timeout
```

Comment:
Place also `timeout = None` outside of any function (after `pid_dir` and `log_dir` assignments) so that it is easier to track that every functions that needs it actually refers to it from global namespace.